proxyclient/hv/trace_dcp.py

```diff
@@ -205,7 +205,7 @@ def handle_cmd(self, hdr, sub, fd):
     "A020": "IOMFB::UPPipeAP_H13P::rt_bandwidth_setup()",
     "A021": "IOMFB::UPPipeAP_H13P::rt_bandwidth_update(IOMFB::AppleRegisterStream*, float, float, bool, bool)",
     "A022": "IOMFB::UPPipeAP_H13P::rt_bandwidth_update_downgrade(IOMFB::AppleRegisterStream*)",
-    "A023": "IOMFB::UPPipeAP_H13P::rt_bandwidth_write_update(IOMFB::AppleRegisterStream*, RealtimeBandwithWritebackBlock, bool)",
+    "A023": "IOMFB::UPPipeAP_H13P::rt_bandwidth_write_update(IOMFB::AppleRegisterStream*, RealtimeBandwidthWritebackBlock, bool)",
```

Comment:
Neither token appears in google... and the only hit in cs is for this line itself.

Reply:
It's a symbol from apple, it appears in some kext.